ansible/playbooks/AdoptOpenJDK_AIX_Playbook/README.md

```diff
@@ -0,0 +1,61 @@
+# Setting up AIX servers
+
+-- Work in Progress -- (WIP)
+
+1) AIX BASE Requirements
+1a) X11 requirements
+# OJDK requires X11 support
+# Additional dependencies are installed by default
+X11.adt.ext
+# X11 Frame Buffers
+X11.vfb
+X11.base.lib
+
+1b) AIX NLS/language support
+# Highly recommended are the following multi-language support files
+# https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/657#issuecomment-454308017
+bos.iconv
+bos.loc.com.CN
+bos.loc.com.JP
+bos.loc.com.utf
+bos.loc.utf.EN_ZA
+bos.loc.iso.en_ZA
+bos.loc.iso.ja_JP
+bos.loc.iso.ko_KR
+bos.loc.iso.zh_CN
+bos.loc.pc
+bos.loc.utf.JA_JP
+bos.loc.utf.KO_KR
+bos.loc.utf.ZH_CN
+bos.loc.utf.ZH_TW
+
+1c) OPENSSL requirements
+The most recent versions of openssh and openssl, in any case
+openssl.base.1.0.2.1600 and later
+
+
+1d) AIX base requirements to be able to use AIX Toolbox
+OSS software packaged by AIX Toolbox will require the following filesets:
+expect
+tk
+tcl
+
+Additionally, the yum setup script (yum.sh) - if used, will require the ftp client. For AIX 7.2 that is a seperate fileset
+bos.net.tcp.ftp
+
+2) ## WIP ## Ansible playbook instructions
+
+3) (build without jenkins-agent)
+Rough Steps: WIP
+* login as jenkins (as the user specific requirements, rbac, capabilities, ulimits are already there)
+* git clone https://github.com/AdoptOpenJDK/openjdk-build.git
+* cd openjdk-build/build-farm
+*  ./make-adopt-build-farm.sh
+
+## The build process should just start. If there are errors, they will need to be dealt with.
+
+4) Additional Notes
+One of the roles (rbac) is needed for some special Java functions - metronome and core dump.
+The setup is performed by the AIX role 'rbac'
+
+## Details of RBAC config to follow ##
```

Comment:
Depending on how long the details are this might be better just inlines into the `main.yml` role itself

Reply:
Likely - for coders that is the right location. And then in a format that it can be bot-parsed and used to update a README file or an FAQ.

First and foremost (imho) is getting the text collected (in a single location) and (re)moved to other locations once it is nearly finished.

Take advantage of the fact that I do not know all these locations - and struggle with having to go to multiple locations to try and figure out what needs to be done.

Another way to think of it would be as a draft for a lesson plan aka tutorial to get started with OpenJDK on AIX. That is a document - in it's own right - and refers to other documents as source. Having the core of those documents - dedicated to a `platform` only makes (imho) that platform more accessible aka easier to use.